go-controller/pkg/ovn/egressip.go

```diff
@@ -534,12 +534,13 @@ func (oc *Controller) addEgressNode(egressNode *kapi.Node) error {
 			klog.Errorf("Re-assignment for EgressIP: unable to retrieve EgressIP: %s from the api-server, err: %v", eIPName, err)
 			return true
 		}
-		if err := oc.addEgressIP(eIP); err != nil {
-			klog.Errorf("Re-assignment for EgressIP: unable to assign EgressIP: %s, err: %v", eIP.Name, err)
+		eIPCopy := eIP.DeepCopy()
```

Comment:
alternatively you could just do `eIP = eIP.DeepCopy()` since I don't think you use the original again. (likewise elsewhere)